TKGame/TKGame.cs

```diff
@@ -265,11 +279,16 @@ protected override void Draw(GameTime gameTime)
 
             Entity.DrawCollisionIntersections(spriteBatch, EntityManager.GetEntities()[0].collisions);
 
+            //Draws Loading Screen Offscreen until needed
+            //spriteBatch.Draw(Art.LoadTexture, transition.rect, Color.White);
+            transition.Draw(spriteBatch);
```

Comment:
This seems like utilized but unseen resource allocation?

Reply:
That was for a different method I was trying. I was basically going to create a bunch of rectangles that slowly decreased in size over time. However that didn't work so I just used a sprite sheet and that comment was left over